### PR TITLE
Pushrq/buildsys win flutter howto

### PR DESCRIPTION
**Pull request by obrasero**
Unintended closing the request #30.
As i dont no how to reopen it, just a new one  hopefully as intended ;)

### CODE REVIEW

**Comment by obrasero** [comment]
@defnax
If you think it could be easier, tell me what to change and i will test with my abslout androidstudio/flutter-newbee-mind and change if this is able to do it :)

**Comment by defnax** [comment]
> @defnax
> If you think it could be easier, tell me what to change and i will test with my abslout androidstudio/flutter-newbee-mind and change if this is able to do it :)

no mean the description can be easier :D 

i used the flutter console instaed going to start menu and search for the cmd
image

i used same path like the flutter instruction
https://flutter.dev/docs/get-started/install/windows#get-the-flutter-sdk
2. Extract the Zip (example C:\Users\<your-user-name>\Documents\flutter_windows_stable_x.x.x-stable.)

File: AndroidStudio-Flutter-setup.md [hunk 1]
```diff
@@ -58,14 +58,28 @@ flutter doctor
 it should now tell all (but maybe chrome) is OK.
 
 # RS mobile: Prepare project
+## Necessary packages
+RS-mobile depends on the following packages to be avail on path-level parallel to RS-mobile itself and with a name actually not matching the their repo names:
+{as of rs-mobile: SHA-1: 8a16a84711388872d455ce2a81dbfe4f8643f63f}
+
+- Repo: https://gitlab.com/Kumaravinash9/retroshare-dart-see.git
+  1. clone it to the same folder-hierarchy level as you will do for rs-mobile
```
Comment (Kumaravinash9) [inline AndroidStudio-Flutter-setup.md:66]: In future we will update the ref of it. So after that we don't have to clone these repo separately

Comment (obrasero) [reply AndroidStudio-Flutter-setup.md:66]: at this future time the section in howto will get also removed.

Comment (Kumaravinash9) [reply AndroidStudio-Flutter-setup.md:66]: okk please make a PR for this on my PR i will merge this. Thnaks

**Comment by defnax** [comment]
now both are needed right, im preparing a howto doc for the docs page
https://gitlab.com/Kumaravinash9/openapi-dart-retroshare-api-wrapper
https://gitlab.com/Kumaravinash9/retroshare-dart-see

File: AndroidStudio-Flutter-setup.md (hunk 1 above)
Comment (obrasero) [reply AndroidStudio-Flutter-setup.md:66]: assuming "my PR" is a shortcut for your Git-Repo.

**Comment by obrasero** [comment]
> now both are needed right
Yes both are needed by yaml.

File: AndroidStudio-Flutter-setup.md (hunk 1 above)
Comment (obrasero) [reply AndroidStudio-Flutter-setup.md:66]: PR on your Reps-master done.

**Comment by defnax** [comment]
i has added for docs page a howto little bit modified the howto, can be improved in future i hope with pics or else
https://github.com/RetroShare/docs/blob/master/docs/developer/retrosharemobile.md
https://retrosharedocs.readthedocs.io/en/latest/developer/retrosharemobile/

**Pull request merged**